src/components/CustomRegisterField/CustomRegisterField.tsx

```diff
@@ -62,7 +62,7 @@ export const CustomRegisterField: FC<Props> = ({ type, value = '', ...props }) =
     case 'datepicker':
       return <DateField {...props} value={value as string} />;
     default:
-      return <Checkbox {...props} checked={value === true} />;
+      return <Checkbox {...props} checked={[true, 'on'].includes(value)} />;
```

Comment:
```suggestion
      return <Checkbox {...props} checked={isTruthyCustomParam(value)} />;
```

Reply:
We already have a helper that does a good job of this. Probably it's no big deal if there are some extra truthy values, but if so, then we can create a copy of the method for INP/JWP params.